godtools/App/Features/UserLessonProgress/Data/UserLessonProgressRepository/Cache/RealmUserLessonProgress.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  RealmUserLessonProgress.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 9/25/24.
+//  Copyright © 2024 Cru. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+class RealmUserLessonProgress: Object {
+    
+    @objc dynamic var lessonId: String = ""
```

Comment:
We can start using the new ```@Persisted``` declaration now that we're using Realm 10.10+.  Some of our realm objects are using it, I have a ticket in the backlog to update the rest of them.  (https://www.mongodb.com/docs/atlas/device-sdks/sdk/swift/model-data/object-models/#persisted-property-attributes)